packages/Textarea/src/Textarea.js

```diff
@@ -55,18 +55,13 @@ function Textarea(props) {
     return function cleanup() {
       window.removeEventListener("resize", resize);
     };
-  });
+  }, []);
 
   React.useEffect(() => {
     if (props.canExpand) {
       resize();
     }
-  }, [props.value]);
-
-  const setRef = node => {
-    textarea = node;
-    props.inputRef(node);
-  };
+  }, [props.value, props.canExpand]);
```

Comment:
Oh actually I meant this effect - if the component is uncontrolled, and typing doesn't result in a re-render this will not run. And the textarea will not expand.